https://issues.redhat.com/browse/ACM-11760 (#6498)
open-cluster-management/rhacm-docs: release_notes/known_issues_governance.adoc

[#postgresql-data-loss]
== PostgreSQL data loss

> If there is data loss to the PostgreSQL server such as restoring to a backup without the latest data, the governance policy propagator on the {product-title-short} hub cluster must be restarted so that it can update the mapping of policies to database IDs. Until you restart the governance policy propagator, new compliance events associated with policies that once existed in the database are no longer recorded. 

Reviewer: Optional: 

make more user focus, see possible rewrite:

```suggestion
If there is data loss to the PostgreSQL server such as restoring to a backup without the latest data, you must restart the governance policy propagator on the {product-title-short} hub cluster so that it can update the mapping of policies to database IDs. Until you restart the governance policy propagator, new compliance events associated with policies that once existed in the database are no longer recorded. 
```

Reviewer: Or, if it's the user how maps the policies to the databases, you could go a step further and rewrite like this: 
```suggestion
If there is data loss to the PostgreSQL server such as restoring to a backup without the latest data, you must restart the governance policy propagator on the {product-title-short} hub cluster. By restarting it, you can update the mapping of policies to database IDs. Until you restart the governance policy propagator, new compliance events associated with policies that once existed in the database are no longer recorded. 
```
